sharding/indexHashedNodesCoordinatorRegistry_test.go

```diff
@@ -42,9 +42,6 @@ func sameValidators(list1 []Validator, list2 []Validator) bool {
 	}
 
 	for i, validator := range list1 {
-		if !bytes.Equal(validator.Address(), list2[i].Address()) {
```

Comment:
should the chances and index be also compared? also in the next comparison

Reply:
done

Reply:
ok